docs/source/usage/implementation/library/rationale.rst

```diff
@@ -175,104 +194,118 @@ Recombining multiple kernel iterations to loop over lines, columns, blocks or an
-  This is not allowed for kernels that should be copyable to a CUDA device because std::is_trivially_copyable requires the kernel to have no non-trivial copy constructors.
-  * a) ... and inherits from the accelerator. 
-    * - The kernel itself has to inherit at least protected from the accelerator to allow the KernelExecutor to access the Accelerator.
-    * - How do accelerator functions called from the kernel (and not within the kernel class itself) access the accelerator methods?
-    Casting this to the accelerator type and giving it as parameter is too much to require from the user.
-  * b) ... and the `operator()` has a reference to the accelerator as parameter.
-    * + This allows to use the accelerator in functions called from the kernel (and not within the kernel class itself) to access the accelerator methods in the same way the kernel entry point function can.
-    * - This would require an additional object (the accelerator) in device memory taking up valuable CUDA registers (opposed to the inheritance solution). At least on CUDA all the accelerator functions could be inlined nevertheless.
- 2. The `operator()` is templated on the accelerator type and has a reference to the accelerator as parameter.
-  * + The kernel can be an arbitrary function object with ALPAKA_FN_HOST_ACC attributes.
-  * + This would allow to instantiate the accelerator independent kernel and set its members before execution.
-  * +/- usable with polymorphic lambdas.
-  * - The `operator()` could be overloaded on the accelerator type but there is no way to specialize the whole kernel class itself, so it always has the same members.
-  * - This would require an additional object (the accelerator) in device memory taking up valuable CUDA registers (opposed to the inheritance solution). At least on CUDA all the accelerator functions could be inlined nevertheless.
+
+#. The kernel is templated on the accelerator type ...
+
+   * (+) This allows users to specialize them for different accelerators. (Is this is really necessary or desired?)
+   * (-) The kernel has to be a class template. This does not allow C++ lambdas to be used as kernels because they are no templates themselves (but only their ``operator()`` can be templated).
+   * (-) This prevents the user from instantiating an accelerator independent kernel before executing it.
+     Because the memory layout in inheritance hierarchies is undefined a simple copy of the user kernel or its members to its specialized type is not possible platform independently.
+     This would require a copy from UserKernel<TDummyAcc> to UserKernel<TAcc> to be possible.
+     The only way to allow this would be to require the user to implement a templated copy constructor for every kernel.
+     This is not allowed for kernels that should be copyable to a CUDA device because std::is_trivially_copyable requires the kernel to have no non-trivial copy constructors.
+   * a) ... and inherits from the accelerator.
```

Comment:
I think, here is an indentation error. `* (+) ...` and `* (-) ...` are not rendered as separate list points.
https://alpaka.readthedocs.io/en/latest/usage/implementation/library/rationale.html#implementation-variants

Same on point `* b)`

Reply:
thanks, fixed (click on your link again to see the result).